packages/near-fast-auth-signer/src/components/AuthCallback/AuthCallback.tsx

```diff
@@ -95,6 +95,20 @@ export const onSignIn = async ({
   searchParams,
   navigate,
   gateway,
+  devicePageCallback,
+}: {
+  accessToken: string;
+  publicKeyFak?: string;
+  public_key_lak: string;
+  contract_id: string,
+  methodNames: string[],
+  // eslint-disable-next-line no-unused-vars
+  setStatusMessage: (arg: string) => void,
```

Comment:
I think if we add a _ before the name, like `_args`, it's the way to tell the linter we won't use this vars. I'm not sure, but I've faced this before too